apps/ensindexer/test/mocks/http-server.ts

```diff
@@ -0,0 +1,17 @@
+import { EnsRainbow } from "@ensnode/ensrainbow-sdk";
+import { http, HttpResponse } from "msw";
+import { setupServer } from "msw/node";
+
+export const server = setupServer(
+  /**
+   * Mock HTTP response for ENSRainbow.
+   *
+   * It is required for during the ENSIndexer bootstrap phase.
+   */
+  http.get("https://customrainbow.com/v1/version", () => {
```

Comment:
instead of using `msw`, it may be preferrable to use vitest's built-in mocks, what that approach tried?

see `packages/ensrainbow-sdk/src/client.test.ts` for an example of stubbing `fetch` and replacing responses with the appropriate expectations

Reply:
i do like how msw integrates at the vitest top-level and allows us to more generally mock external services, but i think per-test mocking is likely the correct approach for now, wdyt?

Reply:
We could still use MSW to have a precise way to mock network responses on per-test level. Will update that approach 👍